BUILD.osx.md

```diff
@@ -45,7 +56,8 @@ Alternatively if using Qt6 from Homebrew, do
 
     cmake -Bbuild -H. -GNinja -DCMAKE_PREFIX_PATH=/usr/local/opt/qt6
```

Comment:
I think this should be symmetrical with the previous statement on setting Qt environment variables.